shared/common-adapters/tab-bar.native.js

```diff
@@ -65,9 +64,6 @@ const TabBarButton = (props: TabBarButtonProps) => {
       <Icon
         type={props.source.icon}
         style={{
-          color: iconColor,
```

Comment:
yeah, we get warnings otherwise. we switched from iconFonts to images so passing a color makes no sense now